scripts/bin/GenerateStaticData.pm

```diff
@@ -704,6 +704,7 @@ sub _texts {
         push @texts, localize('Adjusted High Barrier');
         push @texts, localize('Adjusted Low Barrier');
         push @texts, localize('Corporate Action');
+        push @texts, localize('The final settlement price may differ from the Indicative price.');
 
```

Comment:
@negar-binary , I added this on backend. So you can remove this message from front end